packages/node_modules/nav-frontend-lenker/_lenke.example.js

```diff
@@ -3,6 +3,8 @@ import React from 'react';
 import PT from 'prop-types';
 import 'NavFrontendModules/nav-frontend-lenker-style'; // eslint-disable-line max-len, import/extensions, import/no-unresolved, import/no-extraneous-dependencies
 
+const SvgLinkIcon = () => (<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><g stroke="#000" strokeLinecap="round" strokeLinejoin="round" strokeMiterlimit="10" fill="none"><path d="M14.516 12.5l.086.086c.781.781 2.047.781 2.828 0l3.965-3.964c1.166-1.167 1.166-3.075 0-4.243l-1.758-1.757c-1.166-1.167-3.076-1.167-4.242 0l-3.965 3.964c-.781.781-.781 2.047 0 2.829l.086.085M9.516 11.5l-.086-.085c-.781-.781-2.047-.781-2.828 0l-3.965 3.964c-1.166 1.167-1.166 3.076 0 4.243l1.758 1.757c1.166 1.167 3.076 1.167 4.242 0l3.965-3.964c.781-.781.781-2.047 0-2.829l-.086-.086M8.127 15.889l7.777-7.778" /></g></svg>);// eslint-disable-line max-len
```

Comment:
Denne kan bare flyttes ut til `Lenke.overview.mdx`. Den bør sikkert være med i kode-eksempelet også, bare for å vise folk at den kun inneholder SVG-kode - slik vi gjør her med `<Cog />`: https://design.nav.no/components/knapp#plassering-av-ikon